packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -874,15 +875,15 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     if (this.state.focused) {
       this.setState({ focused: false });
       if (!this.props.multiSelect) {
-        this._submitPendingValue();
+        this._submitPendingValue(event);
       }
     }
   }
 
   /**
    * Submit a pending value if there is one
    */
-  private _submitPendingValue(submittedInput?: KeyCodes) {
+  private _submitPendingValue(submitPendingValueEvent: React.KeyboardEvent<HTMLElement | Autofill> | React.FocusEvent<HTMLInputElement>) {
```

Comment:
Consider wrapping this to multiple lines

Reply:
Traditionally events that are passed to callbacks in fabric use type `any` to get around having to specify all of this

Reply:
Thank goodness, I was debating about type safety vs having a complex parameter, and chose type safety. But looking around now, you're right, there are instances where we just use any.